docs/api/version-history.md

```diff
@@ -13,9 +13,13 @@ keywords: "API, Docker, rcli, REST, documentation"
      will be rejected.
 -->
 
+## v1.44 API changes
+
+[Docker Engine API v1.43](https://docs.docker.com/engine/api/v1.44/) documentation
```

Comment:
```suggestion
[Docker Engine API v1.44](https://docs.docker.com/engine/api/v1.44/) documentation
```

Reply:
DOH! Changed the wrong line 😂 

fixed 👍